packages/component-library/src/MultiLayerMap/createLayers.js

```diff
@@ -1,14 +1,37 @@
 /* eslint-disable no-nested-ternary */
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
I'm trying to be more careful about this error -- does this mean that d3 isn't a component-library dependency, and it's just being hidden because it's imported elsewhere? If that's the case, let's add it as a dependency.

Reply:
I removed that line since I'm not sure why I added it in the first place. The ESLint hook didn't show any errors when I tried to commit the change.